spec/JSON-RPC.md

```diff
@@ -246,6 +246,12 @@ Params:
 
 Return Type: `null` | `H256`
 
+Errors:
+
+| Code | Message | Description |
+|---|---|---|
+| -32602 | Invalid Params | At least one of the parameters is invalid |
```

Comment:
Why don't you add error codes section above and mention only the error codes here instead of repeating the same descriptions over and over again?